### PR TITLE
resolved comments: index, init_db, products, and users

### PR DESCRIPTION
**Pull request by michellemalf**

### CODE REVIEW

File: db/products.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ async function getAllProducts() {
     try {
         const {rows} = await client.query(`
             SELECT * FROM products
-        `, []);
+        `);
```
Comment (Felix-Cadiz) [inline db/products.js:19]: WHERE id = $1

File: db/users.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ async function createUser ({ username, password }) {
         ON CONFLICT (username) DO NOTHING
```
Comment (Felix-Cadiz) [inline db/users.js:11]: INSERT INTO users ( firstname, lastname, email, imgURL, username, password, isAdmin, address)
VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
RETURNING *

**Review by Felix-Cadiz — COMMENTED** [review]
A few fixes are needed in the products and users functions. Comments have been posted for those individual functions.

**Comment by richardedwardbrown3** [comment]
Looked over, nothing wrong. Looks good.

**Pull request merged**